examples/vkcs-lb-loadbalancer-create/main.tf

```diff
@@ -0,0 +1,51 @@
+resource "vkcs_lb_loadbalancer" "loadbalancer" {
+  name = "loadbalancer"
+  vip_subnet_id = "${vkcs_networking_subnet.lb.id}"
+  tags = ["tag1"]
+  timeouts {
+    create = "15m"
```

Comment:
why? probably it's better to change defaults of the ressource in provider code?

Reply:
Removed timeouts